src/lib/expansion/expansion.spec.ts

```diff
@@ -264,6 +264,41 @@ describe('MatExpansionPanel', () => {
         'Expected class to be added after the animation has finished');
   }));
 
+  it('should emit events for body expanding and collapsing animations', fakeAsync(() => {
+    const fixture = TestBed.createComponent(PanelWithContent);
+    fixture.detectChanges();
+    const contentEl = fixture.nativeElement.querySelector('.mat-expansion-panel-content');
+    const beforeExpanded = fixture.componentInstance.panel.beforeExpanded;
+    const afterExpanded = fixture.componentInstance.panel.afterExpanded;
+    const beforeCollapsed = fixture.componentInstance.panel.beforeCollapsed;
+    const afterCollapsed = fixture.componentInstance.panel.afterCollapsed;
+    spyOn(beforeExpanded, 'emit');
```

Comment:
We should subscribe to the `EventEmitter`, rather than stubbing out its `emit` method.